Cargo.lock

```diff
@@ -56,7 +56,7 @@ dependencies = [
  "flate2",
  "futures-core",
  "h2",
- "http 0.2.11",
```

Comment:
So, actix-http is depending on the http 0.2.12 and not the 1.1.0? sadge...

Reply:
yep, that’s why I wanted to use actix-web version, otherwise we had conflicts between our updated version and the one used by actix-web 😔